---
title: "Research"
---

```{r setup, include=FALSE}
knitr::opts_chunk$set(echo = TRUE)
```

<p>

My current research interests lie in the examination of microbiome sequencing data to better understand the various relationships between cancer and microbial agents. I also am interested in looking at various biases in microbiome data and how the use of different bioinformatic tools for the same goal can result in differing conclusions. I am always happy to hear comments and questions regarding my research and highly value collaboration so please feel free to contact me. For more information on my research check out my [CV](CV.html).

<h2> Robust Microbiome analysis </h2>

A myriad of different bioinformatic tools exist for the processing and statistical examination of microbiome data. Of particular interest are the various ways researchers use different tools to adjust for sequencing errors through clustering operational taxonomic units or correcting errors to make amplicon sequence variants. This interest has led me to compare various error correction methods with standard operational taxonomic unit clustering to determine whether biological conclusions would be impacted by tool choice. This paper is published in PeerJ and is freely available [here](https://peerj.com/articles/5364/).

Another step in microbiome examination is the ability to identify taxa that are associated with the researchers outcome of interest. Due to the unique characteristics of microbiome data, there have been a number of different tools developed to identify differentially abundant taxa. Currently in literature there is no gold standard on which tools researchers should apply and as such they are often used interchangeably with one another. To address this myself and colleagues characterized 14 different DA methods over 38 different microbiome datasets to understand how tool choice would impact study conclusions. This paper is published in Nature Communications and is freely available [here](https://www.nature.com/articles/s41467-022-28034-z). 

<h2> Microbiome and Cancer </h2>

There is a strong amount of evidence to indicate that several bacterial and viral agents can cause the development of cancer. To gain a better understand of the various roleâ€™s microbes play in the development of cancer I have been conducting a large population study on the association of microbes found in salivary samples and breast, colon, and prostate cancer. The first publication of this project describes the composition of the oral microbiome in healthy individuals across Atlantic Canada and the various dietary, anthropometric, and sociodemographic features associated with changes in the community membership. This paper is published in mSphere and is freely available [here](https://journals.asm.org/doi/full/10.1128/mSphere.00451-20).

 I am currently working on comparing these healthy controls to individuals that have develop breast, prostate, or colon cancer either before salivary sampling or after salivary sampling. Stay tuned to see the results of this study.


<h2> Other general interests </h2>

<ul>

<li>	Microbial genomics </li>
<li>  Microbial evolution </li>
<li>  Virology </li>
<li>  Machine Learning </li>
<li>  High dimensional data analysis </li>
</ul>



